src/main/java/pl/lodz/p/it/zzpj/botsite/config/security/SecurityConfig.java

```diff
@@ -25,15 +32,30 @@
     @Autowired
     UserDetailsService userDetailsService;
 
+
+    @Bean
+    public CorsConfigurationSource corsConfigurationSource() {
+        CorsConfiguration configuration = new CorsConfiguration();
+        configuration.setAllowedOrigins(Collections.singletonList("*"));
+        configuration.setAllowedMethods(Arrays.asList("HEAD", "GET", "POST", "PUT", "DELETE", "PATCH"));
+        configuration.setAllowCredentials(true);
+        configuration.setAllowedHeaders(Arrays.asList("Authorization", "Cache-Control", "Content-Type"));
+        final UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
+        source.registerCorsConfiguration("/**", configuration);
+        return source;
+    }
+
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         http
                 .csrf().disable()
+                .cors().and()
                 .exceptionHandling()
                 .authenticationEntryPoint(new RestAuthenticationEntryPoint())
                 .and()
                 .authorizeRequests()
                 .antMatchers("/api/admin/**").hasRole(UserRole.ADMIN.getRoleName())
+                .antMatchers("/api/user/").anonymous()
```

Comment:
Anonymous?

Reply:
Tak, na razie to jest endpoint do rejestracji, więc tylko niezalogowani powinni mieć dostęp. Może zmienić adres na /register?

Reply:
To musisz mieć sprecyzowane niestety, bo /api/user mało sugestywne. Może /api/register bardziej?